### DISCUSSION TITLE
Add support for African dust forecast

### DISCUSSION THREAD

**Discussion by trlef19** [breezy-weather/breezy-weather#1090] [Ideas]
Many Europeans countries (and maybe more) get some amount of dust from Sahara, that impacts air quality etc. it would be nice if there was such forecast

**Comment by papjul** [comment]
I believe all air quality sources we support already take that into account?

**Reply by trlef19** [reply]
Yeah, they take into consideration the air pollution and stuff but provide no info about the severity or the duration of the Dust

**Reply by papjul** [reply]
Some sources report hazy condition. We do have a weather icon/background for that.

**Comment by papjul** [comment]
What is expected from the app exactly?

> no info about the severity or the duration of the Dust

You can get that info from the visibility forecast

**Comment by trlef19** [comment]
Maybe something like [that](https://www.meteo.gr/cf-en.cfm?city_id=12) ?

(Tap on dust category)

I just think it would be nice to appear as a weather phenomenon.

**Reply by papjul** [reply]
We can get dust concentration from Open-Meteo, and we could show it with the other air quality data.

Now, I need to find a scale for low/medium/high concentration of dusk.
If I have 60 µg/m³ of dust, I have no idea if it is supposed to be a low concentration or high concentration. The concentration is nice, but the user needs to have an interpretation of the concentration ;)

**Reply by trlef19** [reply]
Maybe [this](https://stratus.meteo.noa.gr/forecast/DUST/dust-1.png) will help

**Reply by papjul** [reply]
Copernicus also has a color scale:
https://atmosphere.copernicus.eu/charts/packages/cams_air_quality/products/europe-air-quality-daily-aggregated-forecast-aerosol-tracers?base_time=202507250000&layer_name=composition_europe_dust_forecast_surface&level=key_0&originating_centre=85_2&projection=opencharts_europe&stream=dame&valid_time=202507252300

However, it has 12 steps with no labels. I still don’t know what it means, especially as it is far from being linear (look at the last step gap!).
A color scale alone (with no labels) is not appropriate for accessibility reasons (for people with color blindness/deficiency), and even for able people, it’s not clear what a light brown or a dark brown mean.

Edit: And another (very different) "label-less" color scale here: [NOAA air quality forecast](https://airquality.weather.gov/?element=dustsdc01&mapcenter=-100.36%252C39.92&mapzoom=5&subregion=CONUS&region=CONUS)

**Reply by trlef19** [reply]
I'm not _really_ sure what the problem is

**Reply by papjul** [reply]
I need a scale. I need to translate a concentration of dust expressed in µg/m³ into an user-friendly text like “Low concentration of dust” or “High concentration of dust”, because nobody knows what “60 µg/m³ of dust” means.

Something like this:
- From 0 µg/m³ to 20 µg/m³ = Very low concentration of dust
- From 20 µg/m³ to 40 µg/m³ = Low concentration of dust
- From 40 µg/m³ to 60 µg/m³ = Medium concentration of dust
- From 60 µg/m³ to 80 µg/m³ = High concentration of dust
- From 80 µg/m³ to 100 µg/m³ = Very high concentration of dust

The values here are totally made up, and it doesn’t have to be a 5 steps scale, but just to give you an idea.

**Reply by trlef19** [reply]
Okay okay I see. The map I gave you has a scale on the side, isn't it enough? If it's not, I'll try to look for a better scale

**Reply by papjul** [reply]
> The map I gave you has a scale on the side, isn't it enough?

No, for the same reasons the color scale from Copernicus isn’t:

> it has 12 steps with no labels. I still don’t know what it means, especially as it is far from being linear (look at the last step gap!).
> A color scale alone (with no labels) is not appropriate for accessibility reasons (for people with color blindness/deficiency), and even for able people, it’s not clear what a light brown or a dark brown mean.

We need a legend for each color meaning.

> I'll try to look for a better scale

When I said “I need to find a scale”, I meant “I” as in I will do the research myself. I'm not particularly asking you to provide it. If you can, it’s great! But otherwise, I was just saying I'm holding off the idea for now, and put the requirements in a comment, so I can remember the research I need to do when I will get back to it ;)
